src/populate_models.py

```diff
@@ -61,16 +63,28 @@
                 'id': obj.id
             }
 
+            if content['type'] == "checkpoint":
+                for i, element in enumerate(obj.content):
+                    if element['type'] == 'utter':
+                        element_obj = Utter.objects.filter(pk=element['id']).first()
+                        obj.content[i]['example'] = random.choice(getattr(element_obj, 'alternatives'))
+                    elif element['type'] == 'intent':
+                        element_obj = Intent.objects.filter(pk=element['id']).first()
+                        obj.content[i]['example'] = random.choice(getattr(element_obj, 'samples'))
+
+                new_obj['content'] = obj.content
+
             contents.append(new_obj)
 
-        
+
         if validate_content(contents):
             story = Story.objects.create(
                 name="Default Name",
                 content=story_content_formatter(contents),
+                is_checkpoint= True if 'is_checkpoint' in each else False,
```

Comment:
Shouldn't it be something like `is_checkpoint=each['fields']['is_checkpoint'] if 'is_checkpoint' in each['fields'] else False`?

Reply:
Yes, after making corrections suggested above. But i'll leave `True` instead of `is_checkpoint=each['fields']`, because everytime this attr appears, it will be True. I made this decision because devs won't need to fill every story with is_checkpoint=False. This way its absence indicates it's false. Ifyou guys prefer the other way, i can fill every story with it.

Reply:
I don't really know if we can assume this "always true" format. It sounds safer to keep it as `is_checkpoint=each['fields']['is_checkpoint']`. It won't be necessary for the dev to put the false values in the remaining stories since there is a check to see if the key exists in the fields dict: `if 'is_checkpoint' in each['fields']`. If the dev omits it, the value will be false.

Reply:
Ok, no problem. I'll change it.